### PR TITLE
Update to latest specs and modules

### PR DESCRIPTION
**Pull request by mikhailshilkov**

### CODE REVIEW

File: discovery/bigqueryreservation_v1beta1.json [hunk 1]
```diff
@@ -994,11 +1039,6 @@
           "description": "If false, any query using this reservation will use idle slots from other reservations within the same admin project. If true, a query using this reservation will execute with the slot capacity specified above at most.",
           "type": "boolean"
         },
-        "maxConcurrency": {
```
Comment (lblackstone) [inline discovery/bigqueryreservation_v1beta1.json:997]: Is this a breaking change?

Comment (mikhailshilkov) [reply discovery/bigqueryreservation_v1beta1.json:997]: Yes, it looks like a breaking change to me

Comment (lblackstone) [reply discovery/bigqueryreservation_v1beta1.json:997]: What's the intended resolution then? Add a changelog note and take the change?

Comment (viveklak) [reply discovery/bigqueryreservation_v1beta1.json:997]: At the moment yes. This may trigger a version bump once we are out of preview.

**Comment by mikhailshilkov** [comment]
Self-merging to avoid conflicts in parallel PRs

**Pull request merged**